src/routes/protos/detail.$id.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect } from 'react';
```

Comment:
sort import

Reply:
There are no relevant rules, and add them to the subsequent PR.